File: tests/fixtures/golden/csharp/Greeter.cs
namespace Golden.Greeting
{
    public class Greeter
    {
        // Method call on `this`.
        public string Greet(string name) => this.Format(name);

        private string Format(string name) => "Hello";
    }

    // Decoy: a second, unrelated `Greet` and `Format`.
    public class LoudGreeter
    {
        public string Greet(string name) => "HELLO";

        private string Format(string name) => "LOUD";
    }
}

File: tests/fixtures/golden/rust/src/greeter.rs
pub struct Greeter;

impl Greeter {
    /// Method call on self.
    pub fn greet(&self, name: &str) -> &'static str {
        self.format(name)
    }

    fn format(&self, _name: &str) -> &'static str {
        "Hello"
    }
}

/// Decoy: a second, unrelated `greet` and `format`.
pub struct LoudGreeter;

impl LoudGreeter {
    pub fn greet(&self, _name: &str) -> &'static str {
        "HELLO"
    }

    fn format(&self, _name: &str) -> &'static str {
        "LOUD"
    }
}
